CC/Sounder/BaseRadioSetUHD.cc

```diff
@@ -27,33 +27,34 @@ BaseRadioSetUHD::BaseRadioSetUHD(Config* cfg) : _cfg(cfg) {
   std::vector<std::string> radio_serial_not_found;
 
   // need to be further modified
+  std::cout << "num of cells are: " << _cfg->num_cells() << std::endl;
   for (size_t c = 0; c < _cfg->num_cells(); c++) {
     size_t num_radios = _cfg->n_bs_sdrs()[c];
     num_bs_antenntas[c] = num_radios * _cfg->bs_channel().length();
+    std::cout << "num bs antennas is " << num_bs_antenntas[c] << std::endl;
```

Comment:
@[asedlmayr](https://github.com/asedlmayr) asedlmayr Pending

Remove this, or call into MLDP_ logging system